great_expectations/experimental/datasources/experimental_base_model.py

```diff
@@ -61,3 +64,36 @@ def yaml(
         if isinstance(stream_or_path, pathlib.Path):
             return stream_or_path
         return stream_or_path.getvalue()
+
+    def __str__(self):
+        return self.yaml()
+
+    ##### Workaround for https://github.com/pydantic/pydantic/issues/935 ####
+    #
+    # Currently there is no way to output properties using pydantics builtin in json and dict
+    # methods. Our current fix does let one use the exclude/include arguments on properties,
+    # we always include them. We could fix this by changing the json and dict signatures
+    # to `def dict(self, *, include, exclude, **kwargs)` and add the implementation.
+    # See this comment for one way to do this:
+    # https://github.com/pydantic/pydantic/issues/935#issuecomment-554378904
+
+    @classmethod
+    def get_properties(cls) -> List[str]:
+        return [
+            prop for prop in cls.__dict__ if isinstance(cls.__dict__[prop], property)
+        ]
+
+    def json(self, **kwargs) -> str:
+        self.__dict__.update(
+            {prop: getattr(self, prop) for prop in self.get_properties()}
+        )
+        attribs = super().json(**kwargs)
+        return attribs
+
+    def dict(self, **kwargs) -> "DictStrAny":
```

Comment:
Do we need quotes here? Could we do `from __future__ import annotations` and avoid this?